src/Backend/Toolchain.ts

```diff
@@ -58,7 +58,7 @@ class Toolchain {
   installed(): Command {
     throw Error('Invalid installed call');
   }
-  run(cfg: string): Command {
+  run(_cfg: string): Command {
```

Comment:
NOTE: It seems that those classes designed as a super class don't need to follow the rule, because they indicaterequired parameters. So I plan to put some underscore and add the rule 

```
{ "argsIgnorePattern": "^_" }
```
in eslint.